.github/workflows/rust.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           - rust: stable
           - rust: beta
           - rust: nightly
-          - rust: 1.48
+          - rust: "1.56.1"
```

Comment:
Always quote numbers in YAML.

See https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell